app/models/user.rb

```diff
@@ -1,6 +1,37 @@
+require "#{Rails.root}/lib/beard_trimmer.rb"
+require "#{Rails.root}/lib/beard_trimmer/with_massage.rb"
+require "#{Rails.root}/lib/beard_shaver.rb"
```

Comment:
Thanks , I have just updated application.rb to load the files in lib directory .